File: backend/src/main/java/com/thanhtam/backend/dto/ChoiceList.java
package com.thanhtam.backend.dto;

import com.thanhtam.backend.entity.Question;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ChoiceList{
    private Question question;
    private List<ChoiceCorrect> choices;
    private Integer point;
    private Boolean isSelectedCorrected;
}

File: front-end-user/src/app/models/choice-list.ts
import {ChoiceCorrect} from './choice-correct';
import {Question} from './question';

export class ChoiceList {
  choices: ChoiceCorrect[];
  question: Question;
  point: number;
  isSelectedCorrected: boolean;


  constructor(choices: ChoiceCorrect[], question: Question, point: number, isSelectedCorrected: boolean) {
    this.choices = choices;
    this.question = question;
    this.point = point;
    this.isSelectedCorrected = isSelectedCorrected;
  }
}
